Denis_Ng/w2_d5/main.js

```diff
@@ -0,0 +1,22 @@
+
+// (1)
+var heading = document.getElementById('heading');
+
+// (2)
+heading.innerHTML = 'Friend';
```

Comment:
answer for question 2 
document.getElementsByTagName('h1')[1].innerHTML = 'Friend';